src/gui/FileBrowser.cpp

```diff
@@ -749,9 +871,9 @@ void FileBrowserTreeWidget::updateDirectory(QTreeWidgetItem * item )
 
 
 
-QPixmap * Directory::s_folderPixmap = NULL;
-QPixmap * Directory::s_folderOpenedPixmap = NULL;
-QPixmap * Directory::s_folderLockedPixmap = NULL;
+QPixmap * Directory::s_folderPixmap = nullptr;
+QPixmap * Directory::s_folderOpenedPixmap = nullptr;
+QPixmap * Directory::s_folderLockedPixmap = nullptr;
```

Comment:
I'd rather see these at the top of the file (kinda standard place to put variable initializers).

Reply:
They're not added in this PR, so I haven't moved them.

Reply:
That doesn't mean you can't move them. :)

Reply:
Yeah, but that's an even more extreme formatting change than adding whitespace, so I'd rather not

Reply:
Also, the way it is now they're defined above the class that uses them. If each class had it's own file, they would end up at the top of the file.

Reply:
That's fair!